DirectEmployers/MyJobs
# [PD-3274] improve totals presentation

## What's this PR do?
Improves the presentation of the totals columns at the bottom of the table in analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of analytics

## How should this be manually tested?
Pull down repository. Go into the analytics app and look at the bottom of the table. You will see the rows have been stretched to fit the full table along with having dividers amongst them

## Any background context you want to provide?
